src_assets/common/assets/web/config.html

```diff
@@ -625,14 +625,74 @@ <h1 class="my-4">Configuration</h1>
         <label for="port" class="form-label">Port</label>
         <input
           type="number"
-          min="0"
-          max="65529"
+          min="1029"
+          max="65514"
           class="form-control"
           id="port"
           placeholder="47989"
           v-model="config.port"
         />
         <div class="form-text">Set the family of ports used by Sunshine</div>
+        <!-- Add warning if any port is less than 1024 -->
+        <div class="alert alert-danger" v-if="(+effectivePort - 5) < 1024">
+          <b>Warning:</b> Ports below 1024 are not recommended!
+        </div>
+        <!-- Add warning if any port is above 65535 -->
+        <div class="alert alert-danger" v-if="(+effectivePort + 21) > 65535">
+          <b>Danger:</b> Ports above 65535 are not available!
+        </div>
+        <!-- Create a port table for the various ports needed by Sunshine -->
+        <table class="table">
+          <thead>
+            <tr>
+              <th scope="col">Description</th>
+              <th scope="col">Protocol</th>
+              <th scope="col">Port</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>HTTPS</td>
+              <td>TCP</td>
+              <td>{{+effectivePort - 5}}</td>
+            </tr>
+            <tr>
+              <td>HTTP</td>
+              <td>TCP</td>
+              <td>{{+effectivePort}}</td>
+            </tr>
+            <tr>
+              <td>Web UI</td>
+              <td>TCP</td>
+              <td>{{+effectivePort + 1}}</td>
+            </tr>
+            <tr>
+              <td>RTSP</td>
+              <td>TCP</td>
+              <td>{{+effectivePort + 21}}</td>
+            </tr>
+            <tr>
+              <td>Video</td>
+              <td>UDP</td>
+              <td>{{+effectivePort + 9}}</td>
+            </tr>
+            <tr>
+              <td>Control</td>
+              <td>UDP</td>
+              <td>{{+effectivePort + 10}}</td>
+            </tr>
+            <tr>
+              <td>Audio</td>
+              <td>UDP</td>
+              <td>{{+effectivePort + 11}}</td>
+            </tr>
+            <tr>
+              <td>Mic (unused)</td>
+              <td>UDP</td>
+              <td>{{+effectivePort + 13}}</td>
+            </tr>
+          </tbody>
+        </table>
```

Comment:
I would avoid to specify which port are what for. People would wrongly assume that they can choose which port might enable, while it's not the case